scio-snowflake/src/main/scala/com/spotify/scio/snowflake/SnowflakeIO.scala

```diff
@@ -186,7 +186,7 @@ final case class SnowflakeTable[T](connectionOptions: SnowflakeConnectionOptions
       .read[T]()
       .fromTable(table)
       .withDataSourceConfiguration(dataSourceConfiguration(connectionOptions))
-      .withStorageIntegrationName(params.storageIntegrationName)
+      .pipe(r => Option(params.storageIntegrationName).fold(r)(r.withStorageIntegrationName))
```

Comment:
isn't this a required parameter on read ?

Reply:
At one point I got an NPE on this one because unset then `null`, but can't find back how I managed to do that...

Reply:
Makes no difference, removing.